gasp-node/rollup/runtime/integration-test/src/nontransfer.rs

```diff
@@ -159,16 +159,16 @@ fn test_market_should_block() {
 		));
 
 		// none can mint
-		assert_err!(
-			pallet_market::Pallet::<Runtime>::mint_liquidity(
-				RuntimeOrigin::signed(foundation),
-				POOL_ID,
-				NATIVE_ASSET_ID,
-				amount,
-				amount,
-			),
-			pallet_market::Error::<Runtime>::NontransferableToken
-		);
+		// assert_err!(
+		// 	pallet_market::Pallet::<Runtime>::mint_liquidity(
+		// 		RuntimeOrigin::signed(foundation),
+		// 		POOL_ID,
+		// 		NATIVE_ASSET_ID,
+		// 		amount,
+		// 		amount,
+		// 	),
+		// 	pallet_market::Error::<Runtime>::NontransferableToken
+		// );
```

Comment:
@mateuszaaa  would not it be better to adapt this test to ensure that it can be done?

Reply:
its fixed in following pr